build/config/siso/brave_siso_config.star

```diff
@@ -0,0 +1,278 @@
+# -*- bazel-starlark -*-
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+"""Siso configuration adjustments for Brave browser."""
+
+load("@builtin//struct.star", "module", "struct")
+
+# Rules to disable remote execution.
+__RULES_TO_DISABLE_REMOTE = [
+    # We have chromium_src mangling which requires additional inputs/outputs
+    # handling. This is not trivial and require careful configuration. Can be
+    # revisited when SISO becomes first class citizen in Brave.
+    "typescript/ts_library",
```

Comment:
cc'ing @fallaciousreasoning for a heads up on this.

Reply:
I'm a little surprised the Lit mangling stuff doesn't work the same way as the `__BRAVE_CHROMIUM_SRC_FILE_EXTENSIONS` stuff - is it because we modify the output of the Chromium build?